shared/fs/row/tlf-type-container.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as I from 'immutable'
+import * as FsGen from '../../actions/fs-gen'
+import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
+import {compose, connect, setDisplayName, type TypedState, type Dispatch} from '../../util/container'
+import TlfType from './tlf-type'
+import * as StateMappers from '../utils/state-mappers'
+
+type OwnProps = $Diff<Types.TlfTypeRowItem, {rowType: 'tlf-type'}> & {
+  routePath: I.List<string>,
+}
+
+const mapStateToProps = (state: TypedState, {name}: OwnProps) => {
+  const _tlfList = Constants.getTlfListFromType(state.fs.tlfs, name)
```

Comment:
might as well just do _tlfList: Constants.getTlf... so its all inline and remove the braces/return